package-lock.json

```diff
@@ -19126,4 +19126,4 @@
       }
     }
   }
-}
+}
```

Comment:
This could be a breaking change (I think it's one too many) would you unstage this file (I think Praveen on Discord left the command).